shapeFunctions/script.js

```diff
@@ -0,0 +1,33 @@
+// Area of Circle
+function getAreaOfCircle(radius) {
+    return Math.PI * radius**2
+}
+const x = getAreaOfCircle(5)
+console.log(x)
+
+
+
+//Function Expression
+// function getAreaOfCircle () {}
+//Function Declaration
+// const getAreaOfCircle = function() {}
+//Arrow Function
+// const getAreaOfCircle = () => {}
+
+
+// Circumference of Circle
+const getCircumferenceofCircle = function(radius) {
```

Comment:
Remember to capitalize all words after the first word in camelCase. You forgot to capitalize the 'of' here. Just a nitpick.